1.hello-world/README.md

```diff
@@ -41,7 +41,7 @@ When we use the Dapr CLI, it creates an environment variable for the Dapr port,
 Next, let's take a look at the ```neworder``` handler:
 
 ```js
-app.post('/neworder', (req, res) => {
+app.post('/v1.0/neworder', (req, res) => {
```

Comment:
this is users method, it doesn't need to have a version in url here. This is applicable to all the changes in this PR where we are adding a route.

The version is only needed on routes /dapr/subscribe

Reply:
@amanbha I thought we needed to version appcallback path for invocation/ subscription/ inputbinding path.